data/hasAdvisoryCycleLane.yml

```diff
@@ -0,0 +1,113 @@
+# Whether the concept of an advisory cycle lane exists, 
+# i.e. a cycle lane whose use is not mandatory and on which cars can cross if they need to
+#
+# For sources, refer to the sources in exclusiveCycleLaneStyle.yml. The legislation does
+# simply not mention such a cycle lane.
+default: false
+
+AD: false
+
+# Straßenverkehrsordnung 1960, I. ABSCHNITT. Allgemeines. § 2. Begriffsbestimmungen. 7a. Mehrzweckstreifen
+# https://www.ris.bka.gv.at/GeltendeFassung.wxe?Abfrage=Bundesnormen&Gesetzesnummer=10011336
+# (it actually does not define how it is marked, only that it is marked)
+AT: true
+
+BA: false
+
+BD: false
+
+# https://nl.wikipedia.org/wiki/Fietsstrook#Fietssuggestiestrook_in_Belgi%C3%AB
+BE: true
+
+# Signalisationsverordnung Art. 74a
+# https://www.fedlex.admin.ch/eli/cc/1979/1961_1961_1961/de#art_74_a
+# (cycle lanes with dashed lines are proper cycle lanes mandatory to use)
+CH: false
+
+# TP 179 - NAVRHOVÁNÍ KOMUNIKACÍ PRO CYKLISTY, 4 Úseky (intravilán), 4.2.2 Skladebnost - Ochranný pruh pro cyklisty
+# https://pjpk.rsd.cz/data/USR_001_2_8_TP/TP_179_2017.pdf
+CZ: true
+
+# Allgemeine Verwaltungsvorschrift zur Straßenverkehrs-Ordnung (VwV-StVO), §12
+# http://www.verwaltungsvorschriften-im-internet.de/bsvwvbund_26012001_S3236420014.htm
+DE: true
+
+# Liiklusmärkide ja teemärgiste tähendused ning nõuded fooridele, lisa 10
+# https://www.riigiteataja.ee/akt/103032011006 + https://www.riigiteataja.ee/aktilisa/1030/3201/1006/MKM_22022011_M12-lisa10.pdf
+# (cycle lanes with dashed lines are proper cycle lanes mandatory to use)
+EE: false
+
+# Real Decreto Legislativo 6/2015, de 30 de octubre, por el que se aprueba el texto refundido de la Ley sobre Tráfico, Circulación de Vehículos a Motor y Seguridad Vial, Art 74-79
+# see https://reconstruccionaccidentestrafico.com/vias-ciclistas-definicion-legal-normativa-y-prioridad-de-paso-respecto-a-otros-vehiculos-y-a-peatones/
+ES: false
+
+FI: false
+
+FR: true # https://duckduckgo.com/?q=chaucidou&t=ffab&iax=images&ia=images
+
+# https://road.cc/content/news/advisory-bike-lines-increase-risk-cyclist-casualties-279553
+GB: true # https://www.google.de/maps/@51.4957567,-0.12058,3a,75y,344.59h,64.38t/data=!3m6!1e1!3m4!1sJTZbilG681QyqAGUN7XfFw!2e0!7i16384!8i8192
+
```

Comment:
```suggestion

# Članak 46.(1), Članak 57.(6)
# https://narodne-novine.nn.hr/clanci/sluzbeni/2008_06_67_2224.html
HR: false
```

In Croatia on the roads we don't have `cycleway=lane`+`cycleway:lane=advisory`, only **mandatory bicycle lanes** _or_ **sharrows** (`cycleway=shared_lane`+`cycleway:lane=pictogram`). The law translates as:

> (1) The driver of a motor vehicle must use only the road, move in the middle of the marked road
traffic lane, i.e. the lane intended for the traffic of the vehicle category to which the vehicle belongs, except in the case danger to life, health and property.
> (6) A driver who, when turning, crosses a bicycle path or a lane that extends along the roadway he is moving on, is obliged to let bicycles moving along the bicycle path or lane in the same or opposite direction.

Reply:
thank you!